src/app/website/shared/contentful/contentful.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Observable } from 'rxjs/Observable';
+
+import { ContentfulConfig } from './contentful.config';
+import { Http, Response } from '@angular/http';
+import { Injectable } from '@angular/core';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/toPromise';
+import { SafeResourceUrl } from '@angular/platform-browser';
+@Injectable()
+export class ContentfulService {
+  private url: SafeResourceUrl;
+  constructor(private http: Http,
+    private contentfulConfig: ContentfulConfig) { }
+
+  buildURL(contentTypeId: string) {
+    // tslint:disable-next-line:max-line-length
```

Comment:
why are you disabling it. If the whole idea of disabling lint is required, why would we use lint in the first place?
please  fix the issue not write a work-around for it